app/customer.py

```diff
@@ -0,0 +1,52 @@
+from app.car import Car
+
+from app.shop import Shop
+
+
+class Customer:
+    def __init__(
+        self,
+        name: str,
+        product_cart: dict,
+        location: list,
+        money: int | float,
+        car: Car
+    ) -> None:
+        self.name = name
+        self.product_cart = product_cart
+        self.location = location
+        self.money = money
+        self.car = car
+
+    def print_money(self) -> None:
+        print(self.money)
+
+    def calculates_the_price(
+        self,
+        list_of_shops: list[Shop],
+        fuel_price: float
+    ) -> list | bool:
+        min_buf_cost = float("inf")
+        shop_class = None
+        for shop in list_of_shops:
+            total_cost = 0
+            total_cost += round((
+                (shop.location[0] - self.location[0]) ** 2
+                + (shop.location[1] - self.location[1]) ** 2

```

Comment:
What about separating distance counting into separate variable? This code will be much more readable.